rllib/core/learner/learner.py

```diff
@@ -1388,22 +1356,36 @@ def _update_from_batch_or_episodes(
 
         self._set_slicing_by_batch_id(batch, value=False)
 
-        # Log all current learning rates of all our optimizers (registered under the
-        # different ModuleIDs).
-        self.metrics.log_dict(
-            {
-                (mid, f"{full_name[len(mid) + 1 :]}_lr"): convert_to_numpy(
-                    self._get_optimizer_lr(self._named_optimizers[full_name])
-                )
-                for mid, full_names in self._module_optimizers.items()
-                for full_name in full_names
-            },
-            window=1,
-        )
+        self._after_update(timesteps)
 
         # Reduce results across all minibatch update steps.
         return self.metrics.reduce()
 
+    # TODO (sven): Document and design this new approach in order to replace
+    #  `additional_update()`.
+    def _after_update(self, timesteps):
```

Comment:
I thought this function needs to be public, but we are using also `_forward_train`, etc. for overriding in the `RLModule` .... so this should be fine.